rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+          {
+            "name": "query0",
+            "query": "FIND aws_account THAT HAS aws_iam THAT !HAS aws_iam_account_password_policy",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-enforce-password-policies-gcp",
+        "description": "M1027 - Password Policies - Set and enforce secure password policies for Google Cloud accounts.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_iam_service_account THAT !RELATES TO Policy WITH displayName ~= 'password'",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-enforce-password-policies-azure",
+        "description": "M1027 - Password Policies - Set and enforce secure password policies for Azure accounts.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND azure_vm THAT HAS azure_policy_state WITH complianceState != 'Compliant' THAT DEFINES azure_policy_definition WITH displayName ~= ('Audit Windows machines that do not have a maximum password age of 70 days' OR 'Audit Windows machines that do not have a minimum password age of 1 day' OR 'Audit Windows machines that do not have the password complexity setting enabled' OR 'Audit Windows machines that do not store passwords using reversible encryption' OR 'Audit Windows machines that allow re-use of the previous 24 passwords' OR 'Audit Windows machines that do not restrict the minimum password length to 14 characters' OR 'Audit Linux machines that allow remote connections from accounts without passwords' OR 'Audit Linux machines that do not have the passwd file permissions set to 0644' OR 'Audit Linux machines that have accounts without passwords')",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-rdp-hijacking-remove-unecessary-remote-desktop-users",
+        "description": "M1047 - Audit - Audit the Remote Desktop Users group membership regularly. Remove unnecessary accounts and groups from Remote Desktop Users groups.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND (User) WITH accountEnabled != true THAT RELATES TO (Group|UserGroup) with displayName ~= 'remote'",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-rdp-disable-rdp-aws",
+        "description": "M1042 - Disable or Remove Feature or Program - Disable the RDP service in aws if it is unnecessary.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND aws_security_group AS sg THAT ALLOWS AS rule Internet WHERE rule.ingress=true AND rule.fromPort <= 3389 AND rule.toPort >= 3389",
```

Comment:
duplicate rule as above

Reply:
remote-services-remote-desktop-protocol-disable-rdp-aws
vs
remote-services-session-hijacking-disable-rdp-aws
vs 
remote-services-session-hijacking-rdp-disable-rdp-aws

This isn't a duplicate, this is the same query applying to a different sub-technique.